dlt/extract/incremental/transform.py

```diff
@@ -218,6 +227,8 @@ def __call__(
             ) from ex
         # new_value is "less" or equal to last_value (the actual max)
         if last_value == new_value:
+            if self.range_start == "open":
+                return None, False, False
```

Comment:
right! maybe this disables deduplication as well...

Reply:
Not completely. Added this check to the `deduplication_disabled` and a test for it, so it gets disabled now